cmd/kube-proxy/app/server_others.go

```diff
@@ -88,7 +88,8 @@ func (s *ProxyServer) createProxier(config *proxyconfigapi.KubeProxyConfiguratio
 		if err != nil {
 			return nil, err
 		}
-		klog.InfoS("NodeInfo", "podCIDR", nodeInfo.Spec.PodCIDR, "podCIDRs", nodeInfo.Spec.PodCIDRs)
+		s.NodeRef.UID = nodeInfo.GetUID()
```

Comment:
weird to have `s.NodeRef.UID` only be correct in this one case (and be explicitly wrong otherwise???)

Reply:
I don't know honestly the difference, I picked the one my IDE returned :)